backend/src/sound.rs

```diff
@@ -37,6 +37,10 @@ impl SoundInterface {
                 }
             })
             .with_ctx(|_| (ErrorKind::SoundError, EXPORT_FILE.to_string_lossy()))?;
+        let instant = Instant::now();
+        while tokio::fs::metadata(&*PERIOD_FILE).await.is_err()
+            && instant.elapsed() < Duration::from_secs(1)
+        {}
```

Comment:
still think this should have a sleep, this is going to thrash too much. 
Like a sleep of 1ms should be fine